bakery/RB_1qb_baking_2.py

```diff
@@ -0,0 +1,88 @@
+from qm import SimulationConfig
+from qm.QmJob import QmJob
+from qm.QuantumMachinesManager import QuantumMachinesManager
+from RB_1qb_configuration import config, pulse_len
+from rb_1_with_table_guide import *
+import matplotlib.pyplot as plt
+
+d_max = 20  # Maximum RB sequence length
```

Comment:
all of this up to line 44 should be refactored into a class that the user inits with d_max, K, the name of the qubit, the name of the ops, and then a class is created which contains everything he needs in order to run the program: K_list, duration tracker, revert op etc.

Reply:
Details of the class to be determined/fixed : 
- how to cope with configuration files (so far we stay in the spirit of the baking and update the input config with all necessary addings, like a black box)
- ensure the writing of the convention in the config of existing operations to be able to play seamlessly all Cliffords (basic operations should for now be given with the right name X, Y, X/2,...